### ISSUE TITLE
A ``requirements`` file for tests.

### ISSUE THREAD

**Issue by fnstudio** [evildmp/VirtualMicroscope#19]
A ``requirements`` file that lists dependencies for tests, eg ``selenium``.

**Comment by tanuck** [comment]
This was moved into the setup.py file instead.

**Comment by fnstudio** [comment]
Yeah, but we might want a dedicated ``requirements.txt`` just to run the tests. Eg, no need to include ``selenium`` in the ``setup.py`` file, I guess.

We might create a ``requirements`` folder in the project root and place a text file there: ``requirements/tests.txt``.

Feel free to upvote/downvote. :)
